pallets/utility/src/lib.rs

```diff
@@ -121,13 +124,15 @@ decl_event! {
     pub enum Event
     {
         /// Batch of dispatches did not complete fully.
-        /// Index of first failing dispatch given, as well as the error.
-        BatchInterrupted(u32, DispatchError),
+        /// Includes a vector of event counts for each dispatch and
+        /// the index of the first failing dispatch as well as the error.
+        BatchInterrupted(EventCounts, ErrorAt),
         /// Batch of dispatches did not complete fully.
-        /// Includes any failed dispatches with their indices and their associated error.
-        BatchOptimisticFailed(Vec<(u32, DispatchError)>),
+        /// Includes a vector of event counts for each call and
+        /// a vector of any failed dispatches with their indices and associated error.
+        BatchOptimisticFailed(EventCounts, Vec<ErrorAt>),
         /// Batch of dispatches completed fully with no error.
```

Comment:
```suggestion
        /// Batch of dispatches completed fully with no error.
        /// Includes a vector of event counts for each dispatch.
```
Just for consistency